system-variables.md

```diff
@@ -596,6 +599,16 @@ Query OK, 0 rows affected (0.09 sec)
 - Default value: `ON`
 - This variable is used to control whether to enable the support for window functions. Note that window functions may use reserved keywords. This might cause SQL statements that could be executed normally cannot be parsed after upgrading TiDB. In this case, you can set `tidb_enable_window_function` to `OFF`.
 
+### `tidb_enforce_mpp` <span class="version-mark">New in v5.1</span>
+
+- Scope: SESSION
+- Default value: `OFF`
+- This variable is used to control whether to ignore the optimizer cost estimation and force queries to be executed using TiFlash's MPP mode. You can set the following values:
+    - `0` or `OFF`, which means MPP mode is not mandatory (default).
```

Comment:
Maybe we can unify the expression of `default`? In `tidb_allow_mpp` it's `by default`, here is `default`.